examples/generative_models/jtnn/README.md

```diff
@@ -20,9 +20,34 @@ These representations can be used for downstream tasks, such as property predict
 > The ZINC database is a curated collection of commercially available chemical compounds 
 prepared especially for virtual screening. (introduction from Wikipedia)
 
-Generally speaking, molecules in the ZINC dataset are more drug-like. We uses ~220,000 
+Generally speaking, molecules in the ZINC dataset are more drug-like. We use ~220,000 
 molecules for training and 5000 molecules for validation. 
 
+### GuacaMol
+
+> The dataset are used by a series of baseline model implementations for the [guacamol](https://github.com/BenevolentAI/guacamol) benchmark for training generative models. The datasets are derived from the ChEMBL 24 database.
```

Comment:
I think you only need to say something like below.

The dataset is used by the [GuacaMol] benchmark of generative models. The dataset is derived from the ChEMBL 24 database.